backend/cmd/server/bootstrap/01-default-resources.ps1

```
@@ -501,7 +503,7 @@ elseif ($response.StatusCode -eq 409) {
 }
 else {
     Log-Error "Failed to create user resource (HTTP $($response.StatusCode))"
-    Log-Error "Response: $($response.Body)"
+    Write-Host "Response: $($response.Body)"
```

PSScriptAnalyzer (1.24.0)

[warning] File '01-default-resources.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)